Fix IP detection by k8s when using calico

adidenko opened the pull request:
- New deb package with missing utility in Ubuntu-14.04, this fixes
  the problem with container IP detection
- Fixed typo in usage() function of build_static_deb.sh script

adidenko committed:
Fix IP detection by k8s when using calico

- New deb package with missing utility in Ubuntu-14.04, this fixes
  the problem with container IP detection
- Fixed typo in usage() function of build_static_deb.sh script

mwhahaha commented on deployment_scripts/puppet/modules/plugin_k8s/manifests/calico_node.pp:
```diff
@@ -32,6 +32,15 @@
+  # a part of util-linux starting from version 2.24, so Ubuntu 14.10 and
+  # newer are OK
+  if $::operatingsystem == 'ubuntu' and $::operatingsystemrelease == '14.04' {
+    package {'nsenter':
```
We probably should just install this as I've noticed warning about this in the kubelet logs even with flannel.

adidenko replied:
OK, will refactor a bit

adidenko committed:
Move 'nsenter' package installation to kubelet

This utility may be needed even for k8s with flannel. So let's
install the package alongside with service that uses ot -'kubelet'
